app/controllers/concerns/export_service.rb

```diff
@@ -435,6 +436,7 @@ def export_work_metadata_as_csv(collection)
       collection.works.includes(:document_sets, :work_statistic, :sc_manifest).reorder(:id).each do |work|
     
         work_users = work.deeds.map{ |d| "#{d.user.display_name}<#{d.user.email}>".gsub('|', '//') }.uniq.join('|')
+        contributors_real_names = work.deeds.map{ |d| "#{d.user.real_name}" }.uniq
```

Comment:
This looks like it'll produce an array, rather than the string we need in the CSV output.  Please join the contributor names using a pipe delimiter as in the previous line.